test/test_helper.rb

```diff
@@ -57,7 +57,7 @@
 
 require 'fixtures/poro'
 
-ActiveSupport.on_load(:active_model_serializers) do
+ActiveSupport.on_load(:action_controller) do
   $action_controller_logger = ActiveModelSerializers.logger
   ActiveModelSerializers.logger = Logger.new(IO::NULL)
```

Comment:
Yeah, I guess since `-      ActiveSupport.run_load_hooks(:active_model_serializers, ActiveModelSerializers)		` was in the railtie, using `action_controller` is just as good